example/properties.yml

```diff
@@ -0,0 +1,12 @@
+---
+dbs:
+  address: 10.244.1.2
```

Comment:
Why is this needed? I see it in their spec but they don't use it. Might need to open an issue for it.